docs/proposals/job_controller.md

```diff
@@ -0,0 +1,226 @@
+**Abstract**
+
+First basic implementation proposal for a Job controller. Several exiting issues were already created regarding that particular subject:
```

Comment:
We want to propose a architecture design, rather than implementation. Additionally I'd rather see that as a complete set of features, at least the most crucial ones we will implement. The implementation doesn't have to cover all from start, but this document should.